conans/client/command.py

```diff
@@ -1533,6 +1533,8 @@ def run(self, *args):
             ret_code = ERROR_GENERAL
             self._user_io.out.error(exc)
         except Exception as exc:
+            import traceback
+            print(traceback.format_exc())
```

Comment:
leftover?

Reply:
Actually @lasote removed this one of his latest PR. 
I find it annoying not to have a trace for generic, not managed Exceptions. It also helps when users are reporting crashes, otherwise we just get some "[Error]: 0" message, or similar.

Reply:
My mistake, Probably I removed it thinking that I was put an extra print. If it was already there it is ok.